openjdk/ProblemList_openjdk11-openj9.txt

```diff
@@ -281,18 +284,18 @@ java/rmi/transport/runtimeThreadInheritanceLeak/RuntimeThreadInheritanceLeak.jav
 
 # jdk_util
 
-java/util/Arrays/TimSortStackSize2.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/Arrays/TimSortStackSize2.java	https://github.com/eclipse/openj9/issues/7086	generic-all
 java/util/Arrays/largeMemory/ParallelPrefix.java https://github.com/eclipse/openj9/issues/4100 linux-ppc64le
 java/util/BitSet/stream/BitSetStreamTest.java https://github.com/eclipse/openj9/issues/4720 linux-all
-java/util/Spliterator/SpliteratorCollisions.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/util/concurrent/atomic/VMSupportsCS8.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/Spliterator/SpliteratorCollisions.java	https://github.com/eclipse/openj9/issues/7090	generic-all
+java/util/concurrent/atomic/VMSupportsCS8.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1347	generic-all
 java/util/concurrent/forkjoin/FJExceptionTableLeak.java	https://github.com/eclipse/openj9/issues/3209	generic-all
-java/util/concurrent/locks/Lock/TimedAcquireLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/concurrent/locks/Lock/TimedAcquireLeak.java	https://github.com/eclipse/openj9/issues/7125	macosx-all,linux-all
 java/util/concurrent/ArrayBlockingQueue/WhiteBox.java      https://github.com/eclipse/openj9/issues/5988    macosx-all
 java/util/logging/LogManager/TestLoggerNames.java https://github.com/eclipse/openj9/issues/4561 generic-all
-java/util/logging/TestLoggerWeakRefLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/logging/TestLoggerWeakRefLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1352	generic-all
```

Comment:
You can add this 1352 link to the ones marked in https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297, close 1352 and use the 1297 link for tests that are permanent excludes due to them being hotspot specific.

Reply:
Yes, I was waiting for clarification if these tests were hotspot specific or not before adding them to the permanent exclude list. So are these tests both to be perma excluded?

Reply:
When i say these i mean this test on all 3 lists and also https://github.com/AdoptOpenJDK/openjdk-tests/issues/1347